lib/msal-common/src/client/BaseClient.ts

```diff
@@ -76,6 +78,8 @@ export abstract class BaseClient {
         // Set the network interface
         this.networkClient = this.config.networkInterface;
 
+        this.setKnownAuthorities(this.config.authOptions.knownAuthorities);
```

Comment:
Should we be checking that validateAuthority is set to true before setting known authorities?

Reply:
There is no longer a "validateAuthority" field. We validate by default. Users can opt out, we will be adding discovery endpoint for AAD and then provide an option to opt out.

Reply:
So for B2C people have to pass in knownAuthorities by default? How will users be able to opt out if there is no validateAuthority field?

Reply:
yes. They will have to set it by default. That will be the future guidance.

Reply:
How will users be able to opt out authority validation?

Reply:
By setting `KnownAuthorities()`. That will be the guidance, either send us a list we need to look up to or we will validate as per the standard route.